releasenotes/notes/add_jetson_check-acf5d948a5875073.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    Add a check for the Nvidia Jetson device family in the IoT Agent.
```

```suggestion
    Add a check for the Nvidia Jetson device family.
```
This will be available on regular builds as well.